docs/pages/router/reference/search-parameters.mdx

```diff
@@ -0,0 +1,200 @@
+- **useGlobalSearchParams**: returns the global URL regardless of the component. This updates on every search param change and might cause components to update extaneously in the background.
+
+Both hooks are typed and accessed the same way, the only difference is how frequently they update. Consider the following example:
+
+## Local vs. Global Search Parameters
+
+The following example demonstrates the difference between `useLocalSearchParams` and `useGlobalSearchParams`.
+
+<FileTree files={['app/_layout.js', 'app/index.js', 'app/[user].js']} />
+
+<Step label="1">
+
+    The Root Layout will be a stack navigator.
+
+    ```js app/_layout.js
+    import { Stack } from 'expo-router';
+
+    export default function Layout() {
+      return <Stack />;
+    }
+    ```
+
+</Step>
+
+<Step label="2">
+
+    The initial route will redirect to the dynamic **app/[user].js** route, with **user=evanbacon**.
+
+    ```js app/index.js
+    import { Redirect } from 'expo-router';
+
+    export default function Route() {
+      return <Redirect href="/evanbacon" />;
+    }
+    ```
+
+</Step>
+
+<Step label="3">
+
+    The dynamic **app/[user]** screen will print out the global and local search params, and enable pushing new instances of the same route.
+
+    ```js app/[user].js
+    import { Text, View } from 'react-native';
+    import { useLocalSearchParams, useGlobalSearchParams, Link } from 'expo-router';
+
+    const friends = ['charlie', 'james']
+
+    export default function Route() {
+      const glob = useGlobalSearchParams();
+      const local = useLocalSearchParams();
+
+      console.log("Local:", local.user, "Global:", glob.user);
+
+      return (
+        <View>
+          <Text>User: {local.user}</Text>
+          {friends.map(friend => (
+            <Link key={friend} href={`/${friend}`}>
+              Visit {friend}
+            </Link>
+          ))}
+        </View>
+      );
+    }
+    ```
+
+</Step>
+
+<Step label="4">
+
+When the app starts, you should see the following log:
+
+<Terminal cmd={['Local: evanbacon Global: evanbacon']} />
+
+Pressing "Visit charlie" will push a new instance of `/[user]` with **user=charlie**, and log the following:
+
+<Terminal
+  cmd={[
+    '# This log came from the new screen',
+    'Local: charlie Global: charlie',
+    '# This log came from the first screen',
+    'Local: evanbacon Global: charlie',
+  ]}
+/>
+
+Pressing "Visit james" will have a similar effect:
+
+<Terminal
+  cmd={[
+    '# This log came from the new, "/james" screen',
+    'Local: james Global: james',
+    '# This log came from the "/evanbacon" screen',
+    'Local: evanbacon Global: james',
+    '# This log came from the "/charlie" screen',
+    'Local: charlie Global: james',
+  ]}
+/>
+
+    **Results:**
+
+    - `useGlobalSearchParams` made the background screens re-render when the search params changed. This can cause performance issues if overused.
```

Comment:
```suggestion
    - `useGlobalSearchParams` made the background screens re-render when the search params changed. It can cause performance issues if overused.
```